docs/source/format/driver_manifests.rst

```diff
@@ -420,7 +422,7 @@ to control which directories will be searched for manifests, with the behavior b
     .. tab-item:: Rust
        :sync: rust
 
-       The ``ManagedDriver`` type has a method ``load_dynamic_from_name`` which takes an optional ``load_flags`` parameter. The flags as a ``u32`` with
+       The ``ManagedDriver`` type has a method ``load_from_name`` which takes an optional ``load_flags`` parameter. The flags as a ``u32`` with
```

Comment:
There's another reference to the Rust driver manager's `load_dynamic_from_name` method higher up (line 175). Do you want to change that one to `load_from_name` too?